backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/CountQuartersAggregator.java

```diff
@@ -13,43 +13,48 @@
 import com.bakdata.conquery.models.externalservice.ResultType;
 import com.bakdata.conquery.models.identifiable.ids.specific.TableId;
 import com.bakdata.conquery.models.query.QueryExecutionContext;
-import com.bakdata.conquery.models.query.queryplan.aggregators.SingleColumnAggregator;
+import com.bakdata.conquery.models.query.queryplan.aggregators.Aggregator;
 import com.bakdata.conquery.models.query.queryplan.clone.CloneContext;
 import it.unimi.dsi.fastutil.ints.IntOpenHashSet;
 import it.unimi.dsi.fastutil.ints.IntSet;
+import lombok.Getter;
 
 /**
  * Count the number of distinct quarters for all events. Implementation is specific for DateRanges
  */
-public class CountQuartersOfDateRangeAggregator extends SingleColumnAggregator<Long> {
+public class CountQuartersAggregator implements Aggregator<Long> {
```

Comment:
Könnte man noch einen ValidityDateAggregator als Abstraction einführen, oder sind die Implementierung in den einzelnen Aggregatoren zu unterschiedlich?

Reply:
Es sieht so aus als hätten alle einen Member für die DateRestriction und der Validity date Column

Reply:
Sie sollten identisch sein hier bietet es sich an

Reply:
aber ist mir gerade zu doof :/